pkgs/development/libraries/libkcapi/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, buildPackages
+  # libkcapi offers more tools one can choose to disable them
+  # for minimalization
```

Comment:
```suggestion
  # libkcapi offers multple tools. They can be disabled for minimization.
```

Reply:
I have changed the comment to what you suggested. Also `enableParallelBuilding` is now also set.